addons/crm/views/utm_campaign_views.xml

```diff
@@ -8,12 +8,18 @@
             <xpath expr="//field[@name='user_id']" position="after">
                 <field name="use_leads"/>
             </xpath>
-            <xpath expr="//div[@id='utm_statistics']" position="inside">
-                <div class="me-3"
+            <xpath expr="//div[hasclass('oe_kanban_bottom_left')]" position="inside">
+                <t t-set="crm_lead_count_label" t-value="record.use_leads.raw_value ? 'Leads' : 'Opportunities'"/>
```

Comment:
I fear this won't be translated, as content extraction is done on tag content and some valid attributes like title, string, ... Idea here would be to do something allowing to extract values, like
```
<t t-if="record.use_leads.raw_value">
    <t t-set="crm_lead_count_label">Leads</t>
<t t-else="">
    <t t-set="crm_lead_count_label">Opportunities</t>
</t>
```
That way 'Leads' and 'Opportunities' are node content and will be correctly extracted and translated (normally ^^ ).